habitat_baselines/common/tensor_dict.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import copy
+from typing import Callable, Dict, Optional, Tuple, Union, overload
+
+import numpy as np
+import torch
+
+TensorLike = Union[
```

Comment:
Typing can be simplified a bit with numbers.Real . Also we don't support integer based arrays for numpy (np.int32, np.int64, np.uint8)

Reply:
I think we do support integer based arrays.  The RGB sensor returns `np.uint8`